pkg/secretsmanager/secretsmanager.go

```diff
@@ -401,6 +410,9 @@ func (sm *secretManagerAWS) CloseClient() {}
 // EnsureSecret saves secret to AWS secret manager
 func (sm *secretManagerAWS) EnsureSecret(ctx context.Context, secretName string, value []byte) error {
 	// get secret ID
+	if binary.Size(value) > awssecretsManagerMaxBytes {
+		return errors.WithStack(fmt.Errorf("unable to write %s to azure vault secret exceeds 65kb", secretName))
```

Comment:
Suggestion: May be better to keep the error messages for Secret Manager and key vault secrets distinct from each other.
s/azure vault secret/secret manager secret/